spec/common-cli-sections.json

```
@@ -248,6 +248,12 @@
         "title": "Delete a function",
         "slug": "supabase-functions-delete",
         "type": "cli-command"
+      },
+      {
+        "id": "supabase-functions-download",
+        "title": "supabase functions download",
```

```suggestion
        "title": "Download a function",
```
We have switched to using human readable titles now. 

Could you also move this to the 2nd position in this list? The sequence is standardised as CRUD

---

done 3db435709